Add a back button in the main options screen.

JohnBergago opened the pull request:
To be consistent with all other screens (except sing screen), we (@DirkHeinke and me) added the Back-Button to the main options screen.

![grafik](https://github.com/user-attachments/assets/2d5ae1c3-c7f4-4f5d-8494-299ba7834ab2)

JohnBergago committed:
Add a back button in the main options screen.

JohnBergago committed:
Merge branch 'develop' into feature/options-back-button

flokuep reviewed (changes requested):
Thanks for your contribution!

For all screen changes, we have a versioning system in place, so when you add or remove elements to/from a screen bump its screen version and add an entry to the according change log.

JohnBergago committed:
Update ScreenOptions Version to 3.
Add entry in Changelog_ScreenOptions.txt. Cleanup previous changelog entries to align with other screen's changelog.

JohnBergago committed:
Merge branch 'develop' into feature/options-back-button

JohnBergago commented:
I updated the version and added a changelog entry. I did reorder the existing changelog, as it was different from the other changelog files.

flokuep reviewed (approved):
Thanks!
